src/components/header/header.tsx

```diff
@@ -0,0 +1,68 @@
+import { Link, NavLink, useNavigate } from '@remix-run/react';
+import classNames from 'classnames';
+import { ROUTES } from '~/router/config';
+import styles from './header.module.scss';
+import { CartIcon } from '../cart-icon/cart-icon';
+import loginIcon from '~/assets/svg/user.svg';
+
+export interface HeaderProps {
+    className?: string;
+}
+
+export const Header = ({ className }: HeaderProps) => {
+    const navigate = useNavigate();
+
+    const menuItemStyle = ({ isActive }: { isActive: boolean }) =>
+        classNames(styles.menuItem, {
+            [styles.active]: isActive,
+        });
+
+    return (
+        <header className={classNames(styles.root, className)}>
+            <section className={styles.topBar}>
+                <Link to={ROUTES.home.to()} className={styles.logo}>
+                    ReClaim
+                </Link>
+                <div className={styles.advertisingText}>
+                    Free shipping on all intl. orders over $35
+                </div>
+                <button
+                    className={styles.shopNowButton}
+                    onClick={() => navigate(ROUTES.products.to('all-products'))}
+                >
+                    Shop Now
+                </button>
+            </section>
+            <section className={styles.navigation}>
+                <div className={styles.empty} />
```

Comment:
> <div className={styles.empty} />

What's this supposed to represent?

Reply:
It keeps menu centred horizontally. 
If I change window size it keeps the same width as actions on the right (login, cart). And it shrinks if menu needs space. 
-------------------------------------------------------------------------------------------------
div.empty 160px |                                      Menu in center                                       |  Log In Cart 160px
-------------------------------------------------------------------------------------------------